certified-connectors/Kanbanize/apiDefinition.swagger.json

```diff
@@ -0,0 +1,910 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Kanbanize",
+    "description": "An official connector working with the Kanbanize API.",
```

Comment:
Could the description please be expanded upon to give users more insight into what the connector will be able to offer? The description should go over the purpose and main value proposition to prospective users.